README.md

```diff
@@ -246,7 +246,7 @@ CBOR decoding/encoding library
 For encoding:
 
 ```c
-#include <cbor_encode.h>
+#include <zcbor_encode.h>
```

Comment:
I would argue that we should stick with "cbor" here since the functionality is pure cbor, and has nothing to do with cddl or higher level functinality.

If there are arguments for changing this its not a deal breaker for me.

Reply:
Past Håkon, we need to prefix with `zcbor` to avoid namespace conflicts!

Reply:
Ah Ok! That makes sense, never mind then :)